functional_tests/jobs/job_artifact_creation.py

```diff
@@ -0,0 +1,11 @@
+import wandb
+
+settings = wandb.Settings()
+settings.update({"disable_git": True, "enable_job_creation": True})
```

Comment:
does this imply that job creation is not enabled by default on runs?

Reply:
Yep, since there's additional overhead and run ending time, it was requested to be opt in.

Reply:
Makes sense - thanks!